Misc/NEWS.d/next/Library/2022-05-20-15-52-43.gh-issue-93010.WF-cAc.rst

```diff
@@ -0,0 +1 @@
+In a very special case, the email package would raise, errors.InvalidHeaderError, which is nonexistent.  errors.InvalidHeaderError has been replaced by errors.InvalidHeaderDefect.
```

Comment:
It might be better to tweak this a bit because the bogus error doesn't really get raised, it gets added to the defect list.  What about:

> In some cases where a header defect was found, the email package tried to add a bogus ``InvalidHeaderError`` to the defect list when it should have been ``InvalidHeaderDefect``.

Reply:
Yikes, my bad! I will make your requested changes now. Thanks!